src/app/components/filebrowseruss/filebrowseruss.component.ts

```diff
@@ -534,14 +559,38 @@ export class FileBrowserUSSComponent implements OnInit, OnDestroy {//IFileBrowse
   }
 
   deleteFile(pathAndName: string): void {
-    this.ussSrv.deleteFile(pathAndName)
+    let deleteSubscription = this.ussSrv.deleteFile(pathAndName)
     .subscribe(
       resp => {
+        this.sendNotification('Editor', 'Deleted: ' + pathAndName);
         this.updateUss(this.path);
       },
-      error => this.errorMessage = <any>error
+      error => {
+        if (error.status == '403') { //Forbidden (TODO: Implement ZSS functionality to return a 403 for permissions problems)
+          this.sendNotification('Editor', 'Incorrect permissions to delete: ' + pathAndName);
+        } else if (error.status == '410') { //Gone (TODO: Implement ZSS functionality to return a 410 for a file/folder that has already been deleted)
+          this.sendNotification('Editor', 'Failed to delete: ' + pathAndName);
+        } else if (error.status == '500') { //Internal Server Error
+          this.sendNotification('Editor', 'Failed to delete: ' + pathAndName);
+        } else {
+          this.sendNotification('Editor', 'Uknown error occured for: ' + pathAndName);
```

Comment:
If you dont know the error, at least show them the status code so they can figure it out.